docker-compose.yml

```diff
@@ -29,6 +29,15 @@ services:
       - localstack
       - oidcprovider
       - statsd
+    depends_on:
```

Comment:
I think we can remove the `link` section. We only have the default network, so everything can talk to everything by default, and th only purpose of the `link` section was to declare dependencies. Since we have `depends_on` now, the `link` section is redundant.